### PR TITLE
Add Work section & About section

### PR DESCRIPTION
**Pull request by astroboyReloaded**
**Work & About Sections**:

- Add grid layout for the Work section.
- Add grid layout for the About section.
- Add all images, icons, and card designs.
- Add functionality to buttons.
- Add footer and social icons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,15 +34,170 @@
...
+          Start collaboration
+        </button>
       </section>
+
+      <!------------ Work Section ------------>
+      <section class="work-container">
+        <figure class="shapes-design-container"></figure>
+        <h1 class="work-title section-title playfair">Projects</h1>
+        <article class="tumbnail-container">
+          <img class="thum-img" src="/images/Project 1.jpg" alt="" />
```
Comment (binhussen) [inline index.html:55]: - Kindly, check your image URL, If you are accessing the image from the same folder use `./` not `/` 👍 👍  , Please do the same for all of your image URLs 😘😘.

File: style.css [hunk 2]
```diff
@@ -141,3 +160,239 @@ header > .nav-bar {
...
+  top: 66%;
+  margin: 0 18px 30px 18px;
+}
+
+.tumbnail-container {
+  display: grid;
+  top: calc(66vw + 1rem);
+  height: fit-content;
+  background-color: #1a2236;
+}
```
Comment (binhussen) [inline style.css:198]: - Kindly, In order to lay out the elements of individual cardcards use Flexbox not grid according to the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).👌👌

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @astroboyReloaded , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Good Job for sticking to the Figma design 👍👍, I think it is better to do the tasks according to the requirements
- kindly, check your image URL and that the image is visible.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by astroboyReloaded** [comment]
**Flex for cards & img URL rename**:

- Changed cards display to flex column.
- Renamed images to unify URL.

File: index.html [hunk 3]
```diff
@@ -30,15 +34,170 @@
...
+        <a href="#" class="start-link poppins"
+          >Get my Resume<i class="up-right-from-square"
+            ><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+              <!--! Font Awesome Pro 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+              <path
+                d="M320 0c-17.7 0-32 14.3-32 32s14.3 32 32 32h82.7L201.4 265.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L448 109.3V192c0 17.7 14.3 32 32 32s32-14.3 32-32V32c0-17.7-14.3-32-32-32H320zM80 32C35.8 32 0 67.8 0 112V432c0 44.2 35.8 80 80 80H400c44.2 0 80-35.8 80-80V320c0-17.7-14.3-32-32-32s-32 14.3-32 32V432c0 8.8-7.2 16-16 16H80c-8.8 0-16-7.2-16-16V112c0-8.8 7.2-16 16-16H192c17.7 0 32-14.3 32-32s-14.3-32-32-32H80z"
+                fill="currentcolor"
+              /></svg></i
+        ></a>
+        <div class="social-icons-container">
```
Comment (Kamba56) [inline index.html:192]: - [Optional] Consider making the list of social icons an unordered list. This is best for HTML semantics.
- Also wrap the icons in an `a` tag that links to the various social media handle. You can use:
```html
<ul>
  <li>
    <a href=""><i ></a>
  </li>
  <li>
    <a href=""><i ></a>
  </li>
</ul>
```

File: style.css [hunk 4]
```diff
@@ -88,56 +94,309 @@ header > .nav-bar {
...
+
+button {
+  color: white;
+  background-color: #1a2236;
+  border: 2px solid white;
+  cursor: pointer;
+}
+
+button:hover {
```
Comment (Kamba56) [inline style.css:231]: - [Optional] Consider adding a pressed state to your buttons to match the figma template. You can use the :active pseudo class

**Review by Kamba56 — APPROVED** [review]
Hi @astroboyReloaded , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (astroboyReloaded) [reply index.html:192]: Thanks for the suggestion! I did!!!

File: style.css (hunk 4 above)
Comment (astroboyReloaded) [reply style.css:231]: Sure thing!

**Comment by astroboyReloaded** [comment]
++I've merged this branch as last reviewed. Working on suggestions++ 
_ Some details have arisen... Putting the icons in a `<ul>` is done and making `buttons` part of a link is done as well! But now I have a couple of issues with the linters; hopefully, I'll have them solved tonight and start fresh by Monday._

**Pull request merged**